pkgs/games/devilutionx/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, SDL2, SDL2_mixer, SDL2_ttf, libsodium, pkg-config }:
+stdenv.mkDerivation rec {
+  version = "unstable-2019-07-28";
+  pname = "devilutionx";
+
+  src = fetchFromGitHub {
+    owner = "diasurgical";
+    repo = "devilutionX";
+    rev = "b2f358874705598ec139f290b21e340c73d250f6";
+    sha256 = "0s812km118qq5pzlzvzfndvag0mp6yzvm69ykc97frdiq608zw4f";
+  };
+
+  NIX_CFLAGS_COMPILE = "-I${SDL2_ttf}/include/SDL2";
+
+  # compilation will fail due to -Werror=format-security
+  hardeningDisable = [ "format" ];
+
+  nativeBuildInputs = [ pkg-config cmake ];
+  buildInputs = [ libsodium SDL2 SDL2_mixer SDL2_ttf ];
+
+  installPhase = ''
```

Comment:
Please add `runHook preInstall` and `runHook postInstall` as the first and last line of `installPhase`.

Reply:
Done